### ISSUE TITLE
Is transition from Exoplayer to Media3 planned?

### ISSUE THREAD

**Issue by schnillerman** [jellyfin/jellyfin-android#1010]
**Describe the feature you'd like**
Transition from soon deprecated exoplayer to Media3 is planned and implemented in jellyfin android.

**Additional context**
See here: https://android-developers.googleblog.com/2023/03/media3-is-ready-to-play.html

**Comment by nielsvanvelzen** [comment]
Yes this is planned but for now it doesn't matter whether we use ExoPlayer or media3 as both will be updated for a while so they share the same codec support etc.

**Comment by jfldsf** [comment]
Hey wondering if yall are aware that Google stops updating exoplayer next month

**Comment by nielsvanvelzen** [comment]
They already stopped updating ExoPlayer a while ago.
